test/arcade/potts/env/location/PottsLocationTest.java

```diff
@@ -1294,31 +1337,154 @@ public void separateVoxels_validLists_updatesCenter() {
+        // Expected voxels in splitVoxels: y >= 1, including the plane where y = 1
+        // (because of randomDoubleZero as RNG)
+        // Expected voxels in locVoxels: y < 1
+        ArrayList<Voxel> splitVoxels = new ArrayList<>();
+        ArrayList<Voxel> locVoxels = new ArrayList<>();
+        for (Voxel voxel : voxels) {
+            if (voxel.y >= 1) {
+                splitVoxels.add(voxel);
+            } else {
+                locVoxels.add(voxel);
+            }
+        }
+
+        PottsLocation loc = new PottsLocationMock(voxels);
+        PottsLocation split = (PottsLocation) loc.split(randomDoubleZero);
+
+        // Verify that the total number of voxels remains the same
+        int totalVoxels = loc.voxels.size() + split.voxels.size();
+        assertEquals(totalVoxels, voxels.size());
+
+        locVoxels.sort(VOXEL_COMPARATOR);
+        loc.voxels.sort(VOXEL_COMPARATOR);
+        splitVoxels.sort(VOXEL_COMPARATOR);
+        split.voxels.sort(VOXEL_COMPARATOR);
+
+        // Verify that the voxels were split correctly
+        assertEquals(locVoxels, loc.voxels);
+        assertEquals(splitVoxels, split.voxels);
     }
     
     @Test
-    public void split_withOffsetsNoDirection_callsMethods() {
-        ArrayList<Integer> offsets = new ArrayList<>(0);
-        offsets.add(0);
-        PottsLocation spy = spy(new PottsLocationMock(voxelListAB));
-        spy.split(RANDOM, offsets);
-        verify(spy).split(eq(RANDOM), eq(offsets), eq(null), eq(0.5));
+    public void split_withOffsetsNoDirection_splitsVoxelsCorrectly() {
+        ArrayList<Voxel> voxels = new ArrayList<>();
+        ArrayList<Voxel> locVoxels = new ArrayList<>();
+        ArrayList<Voxel> splitVoxels = new ArrayList<>();
+
+        
+        // Create a 5x5x5 cuboid of voxels
+        for (int x = 0; x < 5; x++) {
+            for (int y = 0; y < 5; y++) {
+                for (int z = 0; z < 5; z++) {
+                    voxels.add(new Voxel(x, y, z));
+                }
+            }
+        }
+
+        // Only diameter in mock is YZ plane, so plane of division will be ZX plane
+        ArrayList<Integer> offsets = new ArrayList<>();
+        offsets.add(20); // split centered at x = 1
+        offsets.add(20); // split centered at y = 1
+        offsets.add(20); // split centered at x = 1
+
+        // expected locVoxels: y >= 1
+        // expected splitVoxels: y < 1
+        for (Voxel voxel : voxels) {
+            if (voxel.y >= 1) {
+                splitVoxels.add(voxel);
+            } else {
+                locVoxels.add(voxel);
+            }
+        }
+
+        PottsLocation location = new PottsLocationMock(voxels);
+
+        // Call split
+        PottsLocation splitLocation = (PottsLocation) location.split(randomDoubleZero, offsets);
+
+        // Verify the split occurred as expected
+        int totalVoxels = location.voxels.size() + splitLocation.voxels.size();
```

Comment:
I don't think you need the check size part of these tests. If they are not the same size, then it won't pass the content checks.